legal-api/tests/unit/api/test_name_requests.py

```diff
@@ -0,0 +1,36 @@
+# Copyright © 2020 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests to assure the name requests end-point.
+
+Test-Suite to ensure that the /nameRequests endpoint is working as expected.
+"""
+from http import HTTPStatus
+
+
+def test_name_requests_success(client):
```

Comment:
This should get wrapped with an 'integration", you'll need to create one in tests/pytest_marks.py and add it to __init__.py

Reply:
Thanks! updated.